OpenOversight/pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
```

Comment:
I'd like to use a `pyproject.toml` file at some point, as we have way too many configuration files lingering in the repo, imo.